From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 15 Jul 2013 09:53:27 +0200
Subject: Revert "cpufreq: Fix cpufreq regression after suspend/resume"
Patch-mainline: no

This reverts commit c02527487f0c8feb578c0394ad481a97f26d3bd2, upstream
commit f51e1eb63d9c28cec188337ee656a13be6980cfd. It causes regression
during suspend/resume.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/cpufreq/cpufreq_stats.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/cpufreq/cpufreq_stats.c b/drivers/cpufreq/cpufreq_stats.c
index 591b6fb..fb65dec 100644
--- a/drivers/cpufreq/cpufreq_stats.c
+++ b/drivers/cpufreq/cpufreq_stats.c
@@ -349,7 +349,6 @@ static int __cpuinit cpufreq_stat_cpu_callback(struct notifier_block *nfb,
 
 	switch (action) {
 	case CPU_ONLINE:
-	case CPU_ONLINE_FROZEN:
 		cpufreq_update_policy(cpu);
 		break;
 	case CPU_DOWN_PREPARE:
-- 
1.8.3.1

